src/applications/disability-benefits/2346/schemas/definitions/2346UI.js

```diff
@@ -33,7 +33,7 @@ const addressDescription = (
     <p>
       To update your address for all of your VA accounts, you’ll need to go to
       your profile page.{' '}
-      <a href="va.gov/profile">
+      <a href="https:va.gov/profile">
```

Comment:
`https://va.gov/profile`?

Reply:
I originally got `va.gov/profile` from content but it would just append it to the end of the current url of the form.

Reply:
Oh I see what you mean. I forgot the `//`.